source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -887,6 +963,24 @@ class NiDAQmxDriverApiTests : public Test {
     return status;
   }
 
+::grpc::Status perform_bridge_shunt_cal_ex(PerformBridgeShuntCalExResponse & response)
+  {
+    ::grpc::ClientContext context;
+    PerformBridgeShuntCalExRequest request;
+    auto status = stub()->PerformBridgeShuntCalEx(&context, request, &response);
```

Comment:
As in https://github.com/ni/grpc-device/pull/1032, you need to pass parameters.

Add parameters to this function and set them on the request object:
```
    set_request_session_name(request);
    request.set_channel(channel);
    request.set_skip_unsupported_channels(skip_unsupported_channels);
```
(but with the shunt cal parameters)